src/proxy/outbound.rs

```diff
@@ -233,7 +233,7 @@ impl OutboundConnection {
                 .track_outbound(source_addr, dest_addr, req.gateway);
 
         let metrics = self.pi.metrics.clone();
-        let hbone_target = if req.request_type == RequestType::ToServerWaypoint {
+        let hbone_target = if req.protocol == Protocol::HBONE {
```

Comment:
minor fix to make stats properly report for all hbone cases, not just to waypoint